docs/DIDYOUKNOW.md

```diff
@@ -0,0 +1,221 @@
+
+- the Flix `List` module offers more than 95 functions.
+
+- the Flix `String` module offers more than 95 functions.
+
+- the Flix `Foldable` module offers more than 30 functions.
+
+- the Flix standard library follows the convention of "subject-last" to enable
+  pipelining (`|>`).
+
+## Ecosystem
+
+Did you know that:
+
+- Flix has an official Visual Studio Code extension.
+
+- Flix has an official dark theme inspired by Monokai called "Flixify Dark".
+
+- the Flix website (https://flix.dev/) lists the design principles behind Flix.
+
+- Flix has an online playground available at https://play.flix.dev/
+
+- Flix has online API documentation available at https://doc.flix.dev/
+
+- the Flix VSCode extension uses the real Flix compiler.
+
+- the Flix VSCode extension supports auto-complete, jump to definition, hover to
+  show the type and effect of an expression, find all usages, and more.
+
+- the Flix VSCode extension has built-in snippets for type class instances. Try
+  `instance Eq [auto complete]`.
+
+- the Flix VSCode extension supports semantic highlighting.
+
+- the Flix VSCode extension has built-in "code hints" that suggests when lazy
+  and/or parallel evaluation is enabled or inhibited by impurity.
+
+- Flix has community build where Flix libraries can be included in the CI
+  pipeline used to build the Flix compiler.
+
+- Flix has a nascent build system and package manager based on GitHub releases.
+  Today it is possible to build, package, and install Flix packages. Dependency
+  management is in the works.
+
+## Compiler
+
+Did you know that:
+
+- Flix -- by design -- has no compiler warnings, only compiler errors. Warnings
+  can be ignored, but errors cannot be.
+
+- the Flix compiler uses monomorphization hence primitive values are (almost)
+  never boxed.
+
+- the Flix compiler supports incremental and parallel compilation.
+
+- the Flix compiler has more than 28 compiler phases.
+
+- the Flix compiler contains more than 80,000 lines of code.
+
+- the Flix compiler has more than 12,500 manually written unit tests.
+
+- the performance of the Flix compiler is tracked at https://arewefast.flix.dev/
+
+## Other
+
+Did you know that:
+
+- Flix is developed by programming language researchers at Aarhus University
+  (Denmark) in collaboration with researchers at the University of Waterloo
+  (Canada), and at Eberhard Karls University of Tübingen (Germany), and by a
+  growing open source community.
+
+- Several novel aspects of the Flix programming language has been described in
+  the research literature.
+
+- Flix has received grants from the Independent Research Fund Denmark, from
+  Amazon Research, and from the Concordium Foundation.
+
+- Flix has been discussed on Reddit, HackerNews, and LambdaTheUltimate.
+
+- more than 40 people have contributed to the Flix compiler.
+
+- more than 1,500 pull requests have been merged into the Flix compiler.
+
+- Flix used to support uniform function call syntax (UFCS) but we could not
```

Comment:
I think a whole "History" section would be quite interesting

Reply:
Added